01-syntax/02-functions-variables-scope-and-memory/isEven.js

```diff
@@ -4,11 +4,42 @@
 */
 /* global isEven */
 
-// → Your code here.
+function isEven(number) {
+  if (number === 0) {
+    return true;
+  } else if (number === 1) {
+    return false;
+  }
+  return isEven(number - 2);
+}
 
 console.log(isEven(50));
 // → true
 console.log(isEven(75));
 // → false
 console.log(isEven(-1));
-// → ??
+// → Uncaught RangeError: Maximum call stack size exceeded
+
+/* We can to multiply the number to '-1' value.
```

Comment:
What does all that text means ? 
Please clean up source code, if you have some issues to discuss please start discussion in corresponding issue